### PR TITLE
Updates DOI links for two references.

### PR DESCRIPTION
**Pull request by armfazh**

### CODE REVIEW

**Review by alxdavids — APPROVED** [review]

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -64,43 +64,11 @@ informative:
...
-    target: https://doi.org/10.1007/3-540-47721-7_12
-    date: Dec, 2000
-    author:
-      -
-        ins: A. Fiat
-        org: The Weizmann Institute of Science, Israel
-      -
-        ins: A. Shamir
-        org: The Weizmann Institute of Science, Israel
+  FS00: DOI.10.1007/3-540-47721-7_12
```
Comment (cjpatton) [inline draft-irtf-cfrg-voprf.md:67]: Oooh, how does this get rendered in the doc? Do the authors and paper title show up?

Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:67]: the kramdown tools added this feature recently

Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:67]: https://github.com/cabo/kramdown-rfc/wiki/Syntax#references

Comment (cjpatton) [reply draft-irtf-cfrg-voprf.md:67]: Neat!

**Review by chris-wood — APPROVED** [review]

**Pull request merged**